### PR TITLE
Abuse sensor

### PR DESCRIPTION
**Pull request by MaxRobinsonTheGreat**
BYU-PCCL/holodeck#262

### CODE REVIEW

File: Source/Holodeck/Agents/Public/Android.h [hunk 1]
```diff
@@ -78,6 +78,8 @@ class HOLODECK_API AAndroid : public AHolodeckAgent
 		return (void*)CommandArray;
 	}
 
+	float GetAccelerationLimit() override { return 200; }
```
Comment (jaydenmilne) [inline Source/Holodeck/Agents/Public/Android.h:82]: Where did you get these number from? Can you add a comment?

**Review by jaydenmilne — CHANGES_REQUESTED** [review]

File: Source/Holodeck/Agents/Public/Android.h (hunk 1 above)
Comment (MaxRobinsonTheGreat) [reply Source/Holodeck/Agents/Public/Android.h:82]: Well.. I kinda just pulled them out of thin air. 🤷‍♂ I added some comments on why I chose those numbers, but I couldn't find any info on maximum fall distance for roombas or uavs from user manuals, so I just went with intuition.

Comment (jaydenmilne) [reply Source/Holodeck/Agents/Public/Android.h:82]: OK. Did you drop them from different heights and measure the impact? That might be a good proxy, just do some experiments

Comment (MaxRobinsonTheGreat) [reply Source/Holodeck/Agents/Public/Android.h:82]: Yes, the added comments show the maximum height they can be dropped from and I found this by playing around with various drop heights and seeing how much acceleration it caused.

Comment (jaydenmilne) [reply Source/Holodeck/Agents/Public/Android.h:82]: aight

**Review by jaydenmilne — APPROVED** [review]
LGTM, 10/10 great work

**Pull request merged**